app/models/concept-scheme.js

```diff
@@ -19,4 +20,29 @@ export default class ConceptSchemeModel extends Model {
   get createdAt() {
     return this.createdat ?? null;
   }
+
+  asTtlCode() {
+    return `
+      <${this.uri}>
+      ${RDF('type')} ${SKOS('ConceptScheme')} ;
+      ${SKOS('prefLabel')} "${this.label}" .
+    `;
+  }
+
+  async modelWithConceptsAsTtlCode() {
+    const schemeTtlCode = this.asTtlCode();
+    const ttlCodeArray = [];
+
+    for (const concept of await this.getConceptModels()) {
+      ttlCodeArray.push(concept.asTtlCode(this.uri));
+    }
+
+    return [...ttlCodeArray, schemeTtlCode].join(' ');
+  }
```

Comment:
Doubting if this fits in the model context of concept-scheme?

Reply:
we will search for a backend solution later on. For now this will do